docs/src/transformations.md

```diff
@@ -81,7 +81,7 @@ A = Tensor(data, (:i, :j, :k, :l)) #hide
 B = Tensor(rand(2, 2), (:i, :m)) #hide
 C = Tensor(rand(2, 2), (:j, :n)) #hide
 
-tn = TensorNetwork([A, B, C]) #hide
+tn = TensorNetwork(Tensor[A, B, C]) #hide
```

Comment:
```suggestion
tn = TensorNetwork([A, B, C]) #hide
```
We can now maybe remove all these explicit `Tensor` in the `Vector` arguments, so the user does not see any change in the interface. What do you think, @mofeing ?